### ISSUE TITLE
Model Accuracy: 80.60% but give bad result in real data

### ISSUE THREAD

**Issue by hamidfathi1998** [keras-team/keras#13864]
hey I want to forecast [data.xlsx](https://drive.google.com/file/d/1brp9k8wxEOJgg9AGlKh-6koI5A6NZCAm/view?usp=drive_open) this data so after the train model I use model_3.evaluate
for know model accuracy and give 80.60% but when test the other data  I give very less accuracy 
so I want to know what it means that : 
`mse=0.148395, mae=0.385221, mape=101.721314`

my code : 

```
import pandas as pd
from keras.models import Sequential
from keras.layers import Dense
from keras.layers import Dropout
from keras import regularizers
from keras import backend as K
from keras import optimizers
import numpy as np
from sklearn import preprocessing
from sklearn.model_selection import train_test_split

df = pd.read_excel(r'data.xlsx')

dataset = df.values

Y = dataset[:,0]
X = dataset[:,1]


Y = Y.reshape(-1, 1) #returns a numpy array
X = X.reshape(-1, 1) #returns a numpy array

X_train, X_val_and_test, Y_train, Y_val_and_test = train_test_split(X, Y, test_size=0.2,shuffle=False)

mu_x = np.mean(X_train)
mu_y = np.mean(Y_train)

sigma_x = np.std(X_train)
sigma_y = np.std(Y_train)

X_val, X_test, Y_val, Y_test = train_test_split(X_val_and_test, Y_val_and_test, test_size=0.2,shuffle=False)

X_train = (X_train - mu_x)/sigma_x
X_val = (X_val - mu_x)/sigma_x
X_test = (X_test - mu_x)/sigma_x

Y_train = (Y_train - mu_y)/sigma_y
Y_val = (Y_val - mu_y)/sigma_y
Y_test = (Y_test - mu_y)/sigma_y

model_3 = Sequential([
    Dense(3, activation='relu', kernel_regularizer=regularizers.l2(0.01), input_shape=(1,)),
    Dropout(0.3),
    Dense(5, activation='relu', kernel_regularizer=regularizers.l2(0.01)),
    Dropout(0.3),
    Dense(5, activation='relu', kernel_regularizer=regularizers.l2(0.01)),
    Dropout(0.3),
    Dense(3, activation='relu', kernel_regularizer=regularizers.l2(0.01)),
    Dropout(0.3),
    Dense(1, kernel_regularizer=regularizers.l2(0.01)),
])

model_3.compile(optimizer=optimizers.adam(lr=0.0001),
              loss='mse')
hist_3 = model_3.fit(X_train, Y_train,
          batch_size=32,shuffle=False, epochs=100,
          validation_data=(X_val, Y_val))

#------------------------
Epoch 97/100
3252/3252 [==============================] - 0s 151us/step - loss: 1.0044 - val_loss: 0.9967
Epoch 98/100
3252/3252 [==============================] - 1s 157us/step - loss: 1.0031 - val_loss: 0.9963
Epoch 99/100
3252/3252 [==============================] - 0s 147us/step - loss: 1.0024 - val_loss: 0.9961
Epoch 100/100
3252/3252 [==============================] - 0s 147us/step - loss: 1.0032 - val_loss: 0.9956
```

**Comment by KamalChhirang** [comment]
You need to save the value of mu_x, sigma_x, mu_y, sigma y. You need to divide your real-life images with the same value. I think this question should be asked at Stackoverflow.com

**Comment by hamidfathi1998** [comment]
> You need to save the value of mu_x, sigma_x, mu_y, sigma y. You need to divide your real-life images with the same value. I think this question should be asked at Stackoverflow.com

I send data from your email, please look

**Comment by Saduf2019** [comment]
@hamidfathi1998 
Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
